projects/1.20/assets/multipart-machines-farming/mm_farming/lang/zh_cn.json

```
@@ -0,0 +1,76 @@
+{
+  "block.mm_farming.block_controller_farm_fd": "农场控制器",
+  "block.mm_farming.block_controller_farm_phc": "农场控制器",
+  "block.mm_farming.block_controller_farm_plant": "农场控制器",
+  "block.mm_farming.block_controller_farm_tree": "树场控制器",
+  "block.mm_farming.block_crafter_farm": "农场制作台",
+  "block.mm_farming.block_structure_block": "结构方块",
+  "block.mm_farming.str_farm_fd_cabbage": "卷心菜农场",
+  "block.mm_farming.str_farm_fd_onion": "洋葱农场",
+  "block.mm_farming.str_farm_fd_rice": "水稻农场",
+  "block.mm_farming.str_farm_fd_tomato": "番茄农场",
+  "block.mm_farming.str_farm_phc_arid": "干旱菜园农场",
+  "block.mm_farming.str_farm_phc_frost": "严寒菜园农场",
+  "block.mm_farming.str_farm_phc_shaded": "荫蔽菜园农场",
+  "block.mm_farming.str_farm_phc_soggy": "湿润菜园农场",
+  "block.mm_farming.str_farm_phc_tropical": "热带菜园农场",
+  "block.mm_farming.str_farm_phc_windy": "多风菜园农场",
+  "block.mm_farming.str_farm_plant_beetroot": "甜菜农场",
+  "block.mm_farming.str_farm_plant_cactus": "仙人掌农场",
+  "block.mm_farming.str_farm_plant_carrot": "胡萝卜农场",
+  "block.mm_farming.str_farm_plant_glow_berry": "发光浆果农场",
+  "block.mm_farming.str_farm_plant_melon": "西瓜农场",
+  "block.mm_farming.str_farm_plant_nether_wart": "下界疣农场",
+  "block.mm_farming.str_farm_plant_potato": "马铃薯农场",
+  "block.mm_farming.str_farm_plant_pumpkin": "南瓜农场",
+  "block.mm_farming.str_farm_plant_sugar_cane": "甘蔗农场",
+  "block.mm_farming.str_farm_plant_sweet_berry": "甜浆果农场",
+  "block.mm_farming.str_farm_plant_wheat": "小麦农场",
+  "block.mm_farming.str_farm_tree_acacia": "金合欢树场",
+  "block.mm_farming.str_farm_tree_bamboo": "竹子树场",
+  "block.mm_farming.str_farm_tree_birch": "白桦树场",
+  "block.mm_farming.str_farm_tree_cherry": "樱花树场",
+  "block.mm_farming.str_farm_tree_chorus": "紫颂农场",
```

```suggestion
  "block.mm_farming.str_farm_tree_chorus": "紫颂树场",
```
其实是有紫颂树这个称呼的
https://zh.minecraft.wiki/w/%E7%B4%AB%E9%A2%82%E6%A0%91